### ISSUE TITLE
mount -pr is not documented in manual.txt

### ISSUE THREAD

**Issue by cwacha** [dosbox-staging/dosbox-staging#3526]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

latest

### What Operating System are you using?

Windows 11

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Title says it. I noticed this option (which is an excellent solution to mounting from relative locations) while reading the source code. This feature deserves to be documented. 

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Read the documentation in manuals.txt and notice that the -pr option is never described.
2.
3.

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by johnnovak** [comment]
Thanks for raising this @cwacha. I actually did not know about this handy feature 😄 

A proper, fully searchable online User Manual will be released this year on our website. It will be also available for offline use as a zipped HTML archive, _including_ the full-text search feature. That README file will be either removed or stashed away somewhere as a "legacy artifact". As such, we don't want to spend any effort and energy updating the README.

I've raised a separate issue for improving the `MOUNT` command, thanks for bringing our attention to this. 
https://github.com/dosbox-staging/dosbox-staging/issues/3527

Marking the current ticket as "duplicate" for tracking purposes.
